plugins/woocommerce/tests/api-core-tests/tests/products/products-crud.test.js

```diff
@@ -18,118 +23,1084 @@ const { batch } = require( '../../data/shared/batch-update' );
+			// the below has_archives test is currently not working as expected
+			// an issue (https://github.com/woocommerce/woocommerce/issues/34991) 
+			// has been raised and this test can be
+			// updated as appropriate after triage
+			// expect(responseJSON.has_archives).toEqual(true);
+		});
+
+		test('can permanently delete a product attribute', async ({
+			request
+		}) => {
+			// Delete the product attribute.
+			const response = await request.delete(
+				`wp-json/wc/v3/products/attributes/${productAttributeId}`, {
+					data: {
+						force: true,
+					},
+				}
+			);
+			expect(response.status()).toEqual(200);
+
+			// Verify that the product attribute can no longer be retrieved.
+			const getDeletedProductAttributeResponse = await request.get(
+				`wp-json/wc/v3/products/tags/${productAttributeId}`
+			);
+			expect(getDeletedProductAttributeResponse.status()).toEqual(404);
+		});
+
+		test('can batch update product attributes', async ({
+			request
+		}) => {
+			// Batch create 2 product attributes
+			const response = await request.post(
+				`wp-json/wc/v3/products/attributes/batch`, {
+					data: {
+						create: [{
+								name: "Smell"
+							},
+							{
+								name: "Weight"
+							}
+						]
+					}
+				}
+			);
+			const responseJSON = await response.json();
+			expect(response.status()).toEqual(200);
+			expect(responseJSON.create[0].name).toEqual('Smell');
+			expect(responseJSON.create[1].name).toEqual('Weight');
+			const attributeId1 = responseJSON.create[0].id;
+			const attributeId2 = responseJSON.create[1].id;
+
+			// Batch create a new attribute, update an attribute and delete another.
+			const responseBatchUpdate = await request.post(
+				`wp-json/wc/v3/products/attributes/batch`, {
+					data: {
+						create: [{
+							name: "Height"
+						}, ],
+						update: [{
+							id: attributeId1,
+							order_by: "name_num"
+						}],
+						delete: [
+							attributeId2
+						]
+					}
+				}
+			);
+			const responseBatchUpdateJSON = await responseBatchUpdate.json();
+			const attributeId3 = responseBatchUpdateJSON.create[0].id;
+			expect(responseBatchUpdate.status()).toEqual(200);
+
+			const responseUpdatedAttribute1 = await request.get(`wp-json/wc/v3/products/attributes/${attributeId1}`);
+			const responseUpdatedAttribute1JSON = await responseUpdatedAttribute1.json();
+			expect(responseUpdatedAttribute1JSON.order_by).toEqual('name_num');
+
+			// Verify that the deleted product attribute can no longer be retrieved.
+			const getDeletedAttributeResponse = await request.get(
+				`wp-json/wc/v3/products/attributes/${attributeId2}`
+			);
+			expect(getDeletedAttributeResponse.status()).toEqual(404);
+
+			// Batch delete the created attribute
+			await request.post(
+				`wp-json/wc/v3/products/tags/batch`, {
+					data: {
+						delete: [attributeId1, attributeId3]
+					}
+				}
+			);
+		});
+	});
+
+	// test.describe('Product **variations** tests: CRUD', () => {
```

Comment:
Is there a reason we have these tests commented out here? Could we put these in a `skip` block instead, or should we just remove them?

Reply:
These commented out tests should now be removed - thanks for pointing this out!